blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java

```diff
@@ -151,20 +149,6 @@ public void executeRollupForLocatorTriggersExecutionOfRollupRunnable() {
     }
 
     @Test
-    public void executeHistogramRollupForLocatorTriggersExecutionOfHistogramRollupRunnable() {
-
-        // when
-        lfr.executeHistogramRollupForLocator(executionContext, rollupBatchWriter, locators.get(0));
-
-        // then
-        verify(rollupReadExecutor, times(1)).execute(Matchers.<HistogramRollupRunnable>any());
-        verifyNoMoreInteractions(rollupReadExecutor);
-        verify(executionContext, times(1)).incrementReadCounter();
-        verifyNoMoreInteractions(executionContext);
-        verifyZeroInteractions(rollupBatchWriter);
-    }
-
-    @Test
```

Comment:
I see that you removed tests from ``LocatorFetchRunnable`` but I dont see any code being removed from ``LocatorFetchRunnable``.